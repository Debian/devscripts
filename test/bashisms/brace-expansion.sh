#!/bin/sh

echo BASHISM{1..10}
echo BASHISM{1..10..2}
echo BASHISM{a..z}
echo BASHISM{Z..a}
echo BASHISM{a..z..2}
echo {a.._}
echo {a..9}
