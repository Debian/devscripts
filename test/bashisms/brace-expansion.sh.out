possible bashism in bashisms/brace-expansion.sh line 3 (brace expansion, {a..b[..c]}should be $(seq a [c] b)):
echo BASHISM{1..10}
possible bashism in bashisms/brace-expansion.sh line 4 (brace expansion, {a..b[..c]}should be $(seq a [c] b)):
echo BASHISM{1..10..2}
possible bashism in bashisms/brace-expansion.sh line 5 (brace expansion):
echo BASHISM{a..z}
possible bashism in bashisms/brace-expansion.sh line 6 (brace expansion):
echo BASHISM{Z..a}
possible bashism in bashisms/brace-expansion.sh line 7 (brace expansion):
echo BASHISM{a..z..2}
